api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -531,6 +531,11 @@
      */
     Attribute<String> LIBRARY_JARS = new Attribute<>(String2String.getInstance());
 
+    /**
+     * Configuration file for custom SSL keystore to override ssl-server.xml
+     */
+    Attribute<String> CUSTOM_SSL_SERVER_CONFIG = new Attribute<>(String2String.getInstance());
```

Comment:
Yes, you are right although we are doing this only to get custom SSL properties in. I am okay with the name you are suggesting (while we are trying to finalize the change on the Yarn side).